.gitignore

```diff
@@ -7,7 +7,6 @@
 .DS_Store
 .eslintcache
 .nyc_output
-.vscode
```

Comment:
I think .vscode should still be part .gitignore

Reply:
How about [this recommendation](https://github.com/github/gitignore/blob/master/Global/VisualStudioCode.gitignore) from GitHub?
```
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
```

Reply:
Are there any other files that go there? Can `settings` be composed?

Reply:
Per https://stackoverflow.com/questions/32964920/should-i-commit-the-vscode-folder-to-source-control some seem to suggest extensions may occasionally put items in this folder so it's probably best to only allow the four files we need.

As for composing `settings` I didn't find anything suggesting this is currently possible.